package.json

```diff
@@ -11,9 +11,9 @@
   "homepage": "https://github.com/angularclass/angular2-webpack-starter",
   "license": "MIT",
   "scripts": {
-    "build:aot:prod": "npm run clean:dist && npm run clean:aot && webpack --config config/webpack.prod.js  --progress --profile --bail",
+    "build:aot:prod": "npm run clean && npm install && webpack --config config/webpack.prod.js  --progress --profile --bail",
```

Comment:
We always have flexibility to modify this as per our project requirements. Currently I don't see any yarn config in this repo. Anyway we can ask for suggestions.